ais/models.py

```diff
@@ -894,6 +909,8 @@ class AddressSummary(db.Model):
 
     __table_args__ = (
         db.Index('address_summary_opa_account_num_idx', 'opa_account_num', postgresql_using='btree'),
+        db.Index('address_summary_pwd_parcel_id_idx', 'pwd_parcel_id', postgresql_using='btree'),
```

Comment:
generally these go directly in the column definition like

    pwd_parcel_id = db.Column(db.Text, index=True)

would prefer to have sqlalchemy handle the rest